### PR TITLE
Miyoo/Bittboy port

### PR DESCRIPTION
**Pull request by szymor**
In fact, it was tested on Powkiddy V90, but as far as I know it has the same key mapping as the newest Bittboys and PocketGos, so it should be fine.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -64,14 +64,29 @@ target_link_libraries(${BIN_TARGET} PRIVATE
...
 if(TARGET_PLATFORM STREQUAL "retrofw")
   include(CMake/retrofw_defs.cmake)
 elseif(TARGET_PLATFORM STREQUAL "gcw0")
   include(CMake/rg350_defs.cmake)
 elseif(TARGET_PLATFORM STREQUAL "rg350")
   include(CMake/rg350_defs.cmake)
+elseif(TARGET_PLATFORM STREQUAL "miyoo")
+  include(CMake/miyoo_defs.cmake)
+  target_link_libraries(${BIN_TARGET} PRIVATE
+  mpg123
```
Comment (glebm) [inline CMakeLists.txt:78]: Strange that these explicit link targets are needed on Miyoo but nowhere else

Comment (szymor) [reply CMakeLists.txt:78]: It is not strange. The toolchain for Miyoo builds statically, so all dependencies need to be given at compile time.

Comment (glebm) [reply CMakeLists.txt:78]: I'd expect the pkgconfig / cmake config files for these static deps to specify the link requirements corectly (in which case this shouldn't be needed) but I guess they don't.

Comment (szymor) [reply CMakeLists.txt:78]: True, I did not think about it earlier. Anyway, it is a sufficient workaround so let it be.

**Pull request merged**